### PR TITLE
OSM-input-form-survey - Olga, Stenli & Mirela

### PR DESCRIPTION
**Pull request by mirelcac**

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,13 +1,103 @@
...
+  font-size: 20px;
+  margin-bottom: 15px;
 }
+
+.form-container {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  font-family: 'Mooli';
```
Comment (sansan-sakura) [inline src/index.css:18]: I love your font choice!! It seems it fits really well with your design!

File: README.md [hunk 2]
```diff
@@ -6,29 +6,19 @@
...
-### The Problem
+We also structured our components differently, and this was part of what had to be fixed. The last day, there were so many conflicts that we had to go back to a previous version. Perhaps approving pushes should have been added as a rule in this project.
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+Codewise, the biggest challenge was displaying the story after the input forms.
 
 ### View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://app.netlify.com/sites/osm-inputsurvey-vite/configuration/general
```
Comment (sansan-sakura) [inline README.md:21]: This link is connected to a config site, so I think you want to change it.

**Review by sansan-sakura — COMMENTED** [review]
You code is really clean and  well commended. I love the design as it has a real halloween feeling. I also think It is a good idea to create a halloween story as it is almost there :))

File: src/components/Story.jsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+                    The night had transformed from a chilling adventure into a heartwarming celebration. With their new friends, they watched the spectacle unfold, grateful for the memories they had made on this unforgettable Halloween night.
+
+                    As dawn broke and the party came to an end, they bid farewell to their newfound companions and made their way home, hearts aglow with the magic of Halloween. In the end, it was a night filled with both spooky thrills and heartwarming connections—a Halloween tale they would cherish forever.</p>
+            </div>
+
+        </container >
+    )
+}
+
```
[the remark this answers is not in the document]
Comment (elbines) [reply src/components/Story.jsx:25]: I was about to say the same. I find it very hard to read the way it is now. Other than that I like the way your page looks!

File: src/components/Number.jsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  const numberInput = (e) => updateFormData("number", e.target.value)
+  return (
+    <div className="form-container">
+      <div className="input-wrapper">
+        <label><h2>What is your favourite number?</h2></label>
+        <input type="number" id="number-input" value={value} onChange={numberInput} />
+      </div>
+    </div>
+  )
+}
```
Comment (elbines) [inline src/components/Number.jsx:13]: I like that it was possible to both type and select a number by pressing the arrows.

File: src/components/Weather.jsx [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+          <option value="windy">Windy</option>
+          <option value="chilly">Chilly</option>
+        </select>
+      </div>
+    </div>
+  )
+}
+
+Weather.propTypes = {
+  updateFormData: PropTypes.func.isRequired,
```
Comment (elbines) [inline src/components/Weather.jsx:22]: I am curious to why you are using this? I did google, but im not sure if I understand the use. Would like a comment.

**Review by elbines — COMMENTED** [review]
Your project looks really good, and a fun take on it! You did good, especially with struggeling with teamwork.

File: README.md (hunk 2 above)
Comment (mikaeloling) [reply README.md:21]: https://osm-inputsurvey-vite.netlify.app/

File: src/components/Story.jsx [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+export const Story = ({ formData }) => {
+    return (
+        <container>
+            <div className="story-container">
+                <h1>{formData.name}´s Halloween Story!</h1>
+                <p>Once upon a chilling Halloween night, a brave soul named {formData.name} embarked on a spine-tingling adventure. With a lantern in hand and {formData.number} loyal companions by their side, they set out into the {formData.setting}, where the air was thick with anticipation.
+
```
Comment (mikaeloling) [inline src/components/Story.jsx:8]: maybe add a few <p> </p> tags between paragraphs to make it easier to read

File: src/components/MultiStepForm.jsx [hunk 7]
```diff
@@ -0,0 +1,108 @@
+//IMPORT COMPONENTS//
```
Comment (mikaeloling) [inline src/components/MultiStepForm.jsx:1]: A lot of steps and you have a really easy to read code,  nice work!

**Review by mikaeloling — APPROVED** [review]
The end result is really nice, good job guys!

**Pull request closed without merging**